_config.yml

```diff
@@ -8,7 +8,7 @@ federalist_base: "https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cl
 federalist_component_preview: "iframe.html?id="
 federalist_release_prefix: "release-"
 meta:
-  og:image: /img/uswds-logo/lg-black.png
```

Comment:
Thanks for the suggestion. It looks like `og:image` doesn't recognize SVG images (Reference: https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images). This comment encouraged me to check if WebP is currently supported. 
For reference: 
- [Twitter explicitly supports WebP](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image)
- Meta does not explicity support WebP in their docs, but does seem to support WebP in practice. From the [Meta sharing debugger](https://developers.facebook.com/tools/debug) tool:

![image](https://user-images.githubusercontent.com/93996430/175983546-bdb1d7a4-ad98-465a-965f-e7ef426412dd.png)

Because this does not have a direct impact on our performance at this time, I will revert this to the explicitly supported png file.